Release Notes/2016/2016-12-20-cloud-platform-release-notes.md

```diff
@@ -0,0 +1,42 @@
+{{{
+"title": "Cloud Platform - Release Notes: December 20, 2016",
+"date": "12-20-2016",
+"author": "Chris Meyer",
+"attachments": [],
+"contentIsHTML": false
+}}}
+
+### New Features (5)
+* __Simple Backup Service (SBS): Selective File Restore__
+
+	The [Simple Backup Service (SBS)](https://www.ctl.io/simple-backup-service/) has released the ability to selectively restore files and folders. This feature allows users to restore single or multiple paths at the same time for a given server that is being backed up with Simple Backup.<p>
+
+	![SBS Policy Restore Screenshot](../../images/2016-12-20_ReleaseNotes1.png)
+
+
+*	__Network Exchange (NetX)__
+
+  - __DC2 Data Center Operational__ - [NetX](https://www.ctl.io/network-exchange/) is now available in our DC2 data center.
+
+  - __Interface Rate Limiting__ - Users are now able to rate limit an endpoint, virtual or physical, to either 100Mbps, 500Mbps, 1Gbps, or 10Gbps.<p>
+
+  ![NetX Interface Rate Limiting Screenshot](../../images/2016-12-20_ReleaseNotes2.png)
+
+* __CTL.IO__
+
+	- __Online Cloud Estimator__ - [ElasticBox](https://elasticbox.com/) is now available on the [Cloud Estimator](www.ctl.io/estimator/)
+
+	- __Alternative Activation__ - CTL.io now offers an account activation process that does not require a credit card or fraud check. This process also circumvents IP, Cores, Storage, Memory, Bare Metal and Sub Account restrictions associated with WEBC accounts. This form of activation is not publicly available and is targeted at Sales Assisted Customers. Sale Representatives interested in using this feature can apply for an Alternative Activation URL [here](https://cloudhelp.ctl.io/alternative-activation/). CloudHelp and CTL.io Digital Onboarding will be releasing more details about this feature in the weeks to come. If you have more questions in the interim, contact [Rich Dubose](mailto:rich.dubose@ctl.io).
```

Comment:
we should not have personal emails or cloudhelp in public documents.  cloudhelp is for internal consumption only for sellers, we have no interest in this being public facing.  if you need to get a message out to sellers release notes is not the way.